tensorflow_datasets/core/visualization/audio_visualizer.py

```diff
@@ -0,0 +1,68 @@
+""" Audio Visualization
```

Comment:
Could you add a test in `show_example_test.py` to make sure this works ? You can use https://docs.python.org/3/library/unittest.mock.html to make sure the AudioVisualizer is chosen.

Reply:
I tried running show_example_test ( for the imagenet data only ) and encounter the following error : 

ValueError: TFDS has been mocked, but metadata files were not found in /usr/local/lib/python3.6/dist-packages/tensorflow_datasets/testing/metadata/imagenet2012/5.0.0. You should copy the real metadata files, so that the dataset can be loaded properly, or set the data_dir kwarg oftfds.testing.mock_tfds(data_dir=...).

----------------------------------------------------------------------
Ran 2 tests in 0.540s

FAILED (errors=1, skipped=1)